src/composables/multisigAccounts.ts

```diff
@@ -159,13 +174,13 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
           return b.confirmedBy.length - a.confirmedBy.length;
         }
         if (!b.balance.minus(a.balance).isZero()) {
-          return b.balance.minus(a.balance);
+          return b.balance.minus(a.balance).toNumber();
         }
         return new Date(a.createdAt).getTime() - new Date(b.createdAt).getTime();
       });
 
     if (!activeMultisigAccountId.value) {
-      setActiveMultisigAccountId(result[0]?.multisigAccountId);
+      setActiveMultisigAccountId(result[0]?.gaAccountId);
```

Comment:
You were suggesting to stick either to `ga` notation or to `multisig` ones.

Reply:
`gaAccountId` is a thing that comes from the API, so here I'm sticking with the root naming to avoid unnecessary data properties transformation.